libfreerdp/core/settings.c

```diff
@@ -272,53 +315,56 @@ rdpSettings* freerdp_settings_new(DWORD flags)
 	settings->DisableMenuAnims = TRUE;
 	settings->DisableThemes = FALSE;
 	settings->ConnectionType = CONNECTION_TYPE_LAN;
-
 	settings->EncryptionMethods = ENCRYPTION_METHOD_NONE;
 	settings->EncryptionLevel = ENCRYPTION_LEVEL_NONE;
-
 	settings->CompressionEnabled = TRUE;
 
 	if (settings->ServerMode)
-			settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
+		settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
 	else
-			settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
+		settings->CompressionLevel = PACKET_COMPR_TYPE_RDP61;
 
```

Comment:
That is just the default, when using command line arguments you can change that.